apps/website/src/pages/index.astro

```diff
@@ -172,13 +173,19 @@ const currentPage = new URL(Astro.request.url).pathname;
       perspective: 1000px;
       transform-style: preserve-3d;
       @media (max-width: 50em) {
-        display: none;
+        order: 1;
+        width: calc(var(--space-6) * 8);
       }
 
       :global(svg) {
         overflow: visible;
       }
     }
+    .hero-text {
+      @media (max-width: 50em) {
+        order: 2;
```

Comment:
@gretanausedaite What are the changes in this commit ([`b2158c1`](https://github.com/iTwin/iTwinUI-react/pull/785/commits/b2158c15ac611e3d548c29bad3dc24c8ed3f4043)) for? We should not really be using flex order or reverse direction (it's confusing for users as well as for our own code maintenance). I thought our design was already responsive.

Reply:
I updated sizes to fit mobile screens better - I feel like margins were too big before.
I use flex order to arrange hero box items as our designer wanted. Previously we were hiding logo in mobile version.
Removed reverse-direction for consistency.